tools/codegen/src/generator/languages/rust/mod.rs

```diff
@@ -40,9 +40,19 @@ impl super::LanguageGenerator for Generator {
     fn generate<W: io::Write>(writer: &mut W, ast: &ast::Ast) -> io::Result<()> {
         writeln!(writer, "// Generated by Molecule {}", VERSION)?;
         writeln!(writer)?;
+
+        #[cfg(feature = "gen-primitive-types")]
         let code = quote!(
             use molecule::prelude::*;
         );
+
+        #[cfg(not(feature = "gen-primitive-types"))]
+        let code = quote!(
+            use molecule_std as molecule;
+
+            use molecule::prelude::*;
+        );
+
```

Comment:
When `#[cfg(not(feature = "gen-primitive-types"))]`, this happened when an external crate like `ckb` want to use `molecule-std`'s structs or functions